apis/Google.Cloud.Spanner.Admin.Instance.V1/Google.Cloud.Spanner.Admin.Instance.V1/InstanceAdminClient.cs

```diff
@@ -595,7 +595,19 @@ public static InstanceAdminClient Create(gaxgrpc::ServiceEndpoint endpoint = nul
         public static InstanceAdminClient Create(grpccore::Channel channel, InstanceAdminSettings settings = null)
         {
             gax::GaxPreconditions.CheckNotNull(channel, nameof(channel));
-            InstanceAdmin.InstanceAdminClient grpcClient = new InstanceAdmin.InstanceAdminClient(channel);
+            return Create(new grpccore::DefaultCallInvoker(channel), settings);
```

Comment:
Not sure this is a problem, I certainly don't see anything wrong, but for some reason github is not "coloring" this code according to syntax, and the same happens with a lot of the changes below. Changes above, and original code is colored.

Reply:
Scratch that, I just had to reload the page, apparently the CSS hadn't applied entirely.